tests/functional/test_new_resolver_target.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+from pip._internal.cli.status_codes import ERROR, SUCCESS
+from tests.lib.path import Path
+from tests.lib.wheel import make_wheel
+
+
+@pytest.fixture()
+def fake_wheel_tag():
+    return "fakepy1-fakeabi-fakeplat"
```

Comment:
Since it is only used in our test function, we can inline it there instead of using a separate fixture.